packages/fuel-gauge/src/e2e-script.test.ts

```diff
@@ -31,8 +31,8 @@ describe('Live Script Test', () => {
       console.error((e as Error).message);
       console.warn(`
         not enough coins to fit the target?
-        - add assets: https://faucet-beta-5.fuel.network/
-        - check balance: https://fuellabs.github.io/block-explorer-v2/beta-5/#/address/${address}
+        - add assets: https://faucet-devnet.fuel.network/
+        - check balance: https://app.fuel.network/account/${address}/assets/
```

Comment:
Will we use the same URL for the explorer?

Reply:
@arboleya according to @LuizAsFight the URL will remain the same.